### PR TITLE
update dockerfile

### PR DESCRIPTION
**Pull request by butlerx**

### CODE REVIEW

File: network.js [hunk 1]
```diff
@@ -2,10 +2,10 @@
...
-    .client({type: 'web', port: 10303, pin: {role: 'cd-profiles', cmd: '*'}})
-    .client({type: 'web', port: 10303, pin: {role: 'cd-user-profile', cmd: '*'}})
-    .client({type: 'web', port: 10304, pin: {role: 'cd-salesforce', cmd: '*'}})
-    .client({type: 'web', port: 10306, pin: {role: 'cd-events', cmd: '*'}});
+    .client({type: 'web', host: process.env.CD_USERS || 'localhost', port: 10303, pin: {role: 'cd-users', cmd: '*'}})
+    .client({type: 'web', host: process.env.CD_USERS || 'localhost', port: 10303, pin: {role: 'cd-agreements', cmd: '*'}})
+    .client({type: 'web', host: process.env.CD_USERS || 'localhost', port: 10303, pin: {role: 'cd-profiles', cmd: '*'}})
+    .client({type: 'web', host: process.env.CD_USERS || 'localhost', port: 10303, pin: {role: 'cd-user-profiles', cmd: '*'}})
+    .client({type: 'web', host: process.env.CD_EVENTS || 'localhost', port: 10306, pin: {role: 'cd-events', cmd: '*'}})
+    .client({type: 'web', port: 10304, pin: {role: 'cd-salesforce', cmd: '*'}});
```
Comment (Wardormeur) [inline network.js]: remove salesforce then instead of making it a "dead" microservice :)

Comment (butlerx) [inline network.js]: wasn't sure if you wanted it removed in production

Comment (Wardormeur) [inline network.js]: well, even if it wasn't the case, it wouldn't work without host like it's currently :)
Also, we have PRs to remove cp-salesforce, but it didn't get merged, mostly due to a devops side which wasn't controlled (removing cp-salesforce from the current archi).

Comment (butlerx) [inline network.js]: alright will do

**Pull request merged**